### DISCUSSION TITLE
Questions about AppHost

### DISCUSSION THREAD

**Discussion by JessicaKrystal** [microsoft/aspire#2510] [General]
I'm a newcomer.I want to know what to do if I want to update a Project in AppHost. Another point is whether a Project can be restarted on the AppHost dashboard if a Project process is forced to quit

**Comment by davidfowl** [comment]
How are you launching the projects today? Are you using visual studio or running from the command line?

**Reply by JessicaKrystal** [reply]
using visual studio and start without debugging. But after the project is released, can the opening or closing of each service be managed on the dashboard?

**Reply by davidfowl** [reply]
No it can't be. If you launch without debugging, then you can re-build individual projects and visual studio will restart the application without killing the dashboard.

**Comment by nickna** [comment]
In production, the individual projects are deployed into separate containers, which can be restarted from the cloud provider (e.g., Azure, AWS, GCP).

**Reply by davidfowl** [reply]
How is that related to this question??

**Reply by nickna** [reply]
> "after the project is released, can the opening or closing of each service be managed on the dashboard?"

Maybe it isn't. I interpreted this as a question about controlling the production services, which can be performed outside of Aspire's dashboard.
